PowerEditor/src/WinControls/WindowsDlg/WindowsDlg.cpp

```diff
@@ -242,6 +243,7 @@ BEGIN_WINDOW_MAP(WindowsDlgMap)
 			RCTOFIT(IDC_WINDOWS_SORT)
 			RCREST(-1)
 			RCTOFIT(IDCANCEL)
+			RCSPACE(14)

```

Comment:
Remove `RCSPACE(14)` which is not necessary.

Reply:
Removed